torchao/_models/mixtral-moe/model.py

```diff
@@ -225,41 +225,39 @@ def forward(
         y = self.wo(y)
         return y
 
+class MoEFeedForward(nn.Module):
```

Comment:
is this the same code as in the original mixtral model definition? if not, do we have a test testing for numerical equivalency?

Reply:
yes its the original code. We don't have tests for numerical equivalence because i don't think that implementation is actually a source of truth. It can't actually do batchsize 8 due to the crazy dim join going on.